### DISCUSSION TITLE
Registry set via RTR

### DISCUSSION THREAD

**Discussion by Sl1m007** [CrowdStrike/psfalcon#226] [Q&A] [answered]
Has anyone come across setting a registry with value via PSFalcon?
If so could you should your script with me, please?

**Comment by bk-cs** [comment, marked as answer]
From the [US-1 RTR documentation](https://falcon.crowdstrike.com/documentation/71/real-time-response-and-network-containment#rtr_commands):

```
Invoke-FalconRtr -Command 'reg set' -Argument "'HKLM\SOFTWARE\TestKey' TestValue -ValueType=REG_SZ -Value=MyStringValue" -HostId <id>, <id>
```
